projects/angular/src/layout/breadcrumbs/_breadcrumbs.clarity.scss

```
@@ -20,19 +20,18 @@
     .clr-breadcrumb-item {
       display: flex;
       align-items: center;
-      min-height: mixins.baselinePx(24);
-      font-family: breadcrumb-variables.$clr-breadcrumb-font-family;
-      color: breadcrumb-variables.$clr-breadcrumb-link-color;
       @include mixins.generate-typography-token('SECONDARY-13-RG-CPT');
 
       a {
         text-decoration: none;
+        text-align: center;
+        line-height: mixins.baselinePx(24);
+      }
 
-        &::after {
-          content: '/';
-          margin: 0 breadcrumb-variables.$clr-breadcrumb-item-space;
-          color: breadcrumb-variables.$clr-breadcrumb-active-color;
-        }
+      &::after {
```

You can have the `:after` rule on one place.

```suggestion
      &:not(:last-child):after {
```